Core/DISET/private/Transports/BaseTransport.py

```diff
@@ -115,6 +114,7 @@ def _write( self, buffer ):
     return S_OK( self.oSocket.send( buffer ) )
 
   def sendData( self, uData, prefix = False ):
+    from DIRAC.Core.Utilities import DEncode
```

Comment:
Before we forget why, I think it's worth adding a note to this line that explains WHY it is imported here and not at the beginning of the module